src/metabase/xrays/automagic_dashboards/core.clj

```diff
@@ -560,7 +560,8 @@
            :cards dashcards)))
 
 (def ^:private ^:const ^Long max-related 8)
-(def ^:private ^:const ^Long max-cards 30)
+(def ^:private ^:const ^Long max-cards 15)
```

Comment:
I think you should rename this one too in order to clarify the context, e.g. `max-cards-section` perhaps.

Reply:
I'm not too sure about that unfortunately. This is like a default limit for everything and I'm not understanding the code well enough.

Reply:
Fair enough - that makes two of us.